README.md

```diff
@@ -577,20 +443,234 @@ anything. If a company exits at a very high valuation relative to what was expec
 liquidation preferences are not going to matter;
 in that case, everyone will own valuable stock.
 
-## Equity Compensation Details
+## Equity Compensation Basics
 
-In this section we’ll lay out the differences, benefits, and drawbacks of common types of
-granted equity.
-We explain stock options and restricted stock units (RSUs), which the most common forms of
-equity compensation.
-If you want to jump ahead, RSUs are more common for larger companies and options more
+### What is compensation?
+
+🄳 **Compensation** is what you get in exchange for working for a company.
```

Comment:
Wondering if "compensation" and this gloss of "equity" should follow the defining occurrence of equity and of stock options, as they do now? This section seems to cover more basic material than what precedes it. Does "Why give equity compensation" make the most sense coming _after_ more advanced stuff like classes of stock? Seems like it could be more of the "What is a company" type information.